pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -234,7 +239,21 @@ private void startReceivingMessages(List<ConsumerImpl<T>> newConsumers) {
         }
     }
 
+    private void waitForSeekComplete() {
+        while (duringSeek.get()) {
+            if (log.isDebugEnabled()) {
+                log.debug("[{}] [{}] is during seek processing", topic, subscription);
+            }
+            try {
+                Thread.sleep(seekCompleteCheckTicketMs);
```

Comment:
Thread.sleep shouldn't be run on any Netty threads. I think this solution has to be revisited. Which type of thread pool is this code running on?

Reply:
It's not a Netty thread. 
> Bookkeeper code includes Thread.sleep, but we should avoid that in Pulsar code base.
change to schedule later.